netstandard/tools/targets/NETStandard.Library.targets

```diff
@@ -137,4 +139,6 @@
       <Output TaskParameter="TouchedFiles" ItemName="FileWrites" />
     </Touch>
   </Target>
+
+  <Import Project="NETStandard.Library.Trimming.targets" Condition="'$(_SupportsTrimming)' == 'true' AND '$(TrimUnusedDependencies)' == 'true'" />
```

Comment:
Yeah I missed the fact that it was only set in one case due to the diff being collapsed.